02-peer-protocol.md

```diff
@@ -938,6 +960,38 @@ previously sent.
 A receiving node MAY ignore spurious message retransmission, or MAY
 fail the channel if they occur.
 
+### Rationale
+
+The effect of requirements above are that the opening phase is almost
+atomic: if it doesn't complete, it starts again.  The only exception
+is where the `funding_signed` message is sent and not received: in
+this case, the funder will forget the channel and presumably open
+a new one on reconnect; the other node will eventually forget the
+original channel due to never receiving `funding_locked`.
```

Comment:
If you never receive funding_locked, you can't use the channel, and you'll presumably eventually time out.  You're right though, I keep forgetting about the pushMsat case; you do need to care here.

I'm starting to hate pushMsat :(  It means I can pushMsat to you, get the funding tx then abort, and you'll have to remember it just in case I decide to pay you...